.travis.yml

```diff
@@ -1,41 +1,24 @@
-#language: shell
+language: bash
 
-# Required environment variables from repository settings
```

Comment:
Answer from SecuriteInfo.com:

Hello Viktor,

> We are developing this updater for clamav:
> https://github.com/extremeshok/clamav-unofficial-sigs

Would you be so kind and give us a Pro licence for free?
I think the correct address is: admin@extremeshok.com
Could admin@extremeshok.com register an account @securiteinfo.com ?

> We will use it only in TravisCI builds.

The ID of pro account should never be published. Ok ?
BTW, what is TravisCI ?

-- 
Cordialement,

Arnaud Jacques
Gérant de SecuriteInfo.com